aws-amp/adot/example-configs/kubecost-adot-scrape-config.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: aws-adot
+    component: adot-collector-conf
+  name: adot-collector-conf
+data:
+  adot-collector-config: |
+    receivers:
+      prometheus:
+        config:
+          global:
+            external_labels:
+              cluster: YOUR_CLUSTER_NAME_HERE
+            scrape_interval: 60s
+            scrape_timeout: 50s
+          scrape_configs:
+          - job_name: kubecost-amp
+            honor_labels: true
+            dns_sd_configs:
+            - names:
+              - YOUR_NAMESPACE-cost-analyzer.YOUR_NAMESPACE
+              type: 'A'
+              port: 9003
+          # optionally add kubecost network costs scrape job:
+          - job_name: kubecost-networking
+            kubernetes_sd_configs:
+              - role: pod
+            relabel_configs:
+            # Scrape only the the targets matching the following metadata
+              - source_labels: [__meta_kubernetes_pod_label_app_kubernetes_io_instance]
+                action: keep
+                regex:  kubecost
+              - source_labels: [__meta_kubernetes_pod_label_app_kubernetes_io_name]
+                action: keep
+                regex:  network-costs
+
+          - job_name: kubernetes-nodes-cadvisor
+            bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+            kubernetes_sd_configs:
+            - role: node
+            relabel_configs:
+            - action: labelmap
+              regex: __meta_kubernetes_node_label_(.+)
+            - replacement: kubernetes.default.svc:443
+              target_label: __address__
+            - regex: (.+)
+              replacement: /api/v1/nodes/$$1/proxy/metrics/cadvisor
+              source_labels:
+              - __meta_kubernetes_node_name
+              target_label: __metrics_path__
+            scheme: https
+            tls_config:
+              ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+              insecure_skip_verify: true
+
```

Comment:
Should we also be specifying the recording rules in these examples? Or are they no longer required?

Reply:
Not required and also some concern with how it would handle multi-cluster with our current rule.